solidity/contracts/bridge/BridgeGovernance.sol

```diff
@@ -0,0 +1,1542 @@
+contract BridgeGovernance is Ownable {
+    using BridgeGovernanceParameters for BridgeGovernanceParameters.DepositData;
+    using BridgeGovernanceParameters for BridgeGovernanceParameters.RedemptionData;
+    using BridgeGovernanceParameters for BridgeGovernanceParameters.MovingFundsData;
+    using BridgeGovernanceParameters for BridgeGovernanceParameters.WalletData;
+    using BridgeGovernanceParameters for BridgeGovernanceParameters.FraudData;
+
+    BridgeGovernanceParameters.DepositData internal depositData;
+    BridgeGovernanceParameters.RedemptionData internal redemptionData;
+    BridgeGovernanceParameters.MovingFundsData internal movingFundsData;
+    BridgeGovernanceParameters.WalletData internal walletData;
+    BridgeGovernanceParameters.FraudData internal fraudData;
+
+    Bridge internal bridge;
+
+    // Array is used to mitigate the problem with the contract size limit.
+    // governanceDelays[0] -> governanceDelay
+    // governanceDelays[1] -> newGovernanceDelay
+    // governanceDelays[2] -> governanceDelayChangeInitiated
+    uint256[3] public governanceDelays;
+
+    uint256 public bridgeGovernanceTransferChangeInitiated;
+    address internal newBridgeGovernance;
+
+    event GovernanceDelayUpdateStarted(
+        uint256 newGovernanceDelay,
+        uint256 timestamp
+    );
+    event GovernanceDelayUpdated(uint256 governanceDelay);
+
+    event BridgeGovernanceTransferStarted(
+        address newBridgeGovernance,
+        uint256 timestamp
+    );
+    event BridgeGovernanceTransferred(address newBridgeGovernance);
+
+    event DepositDustThresholdUpdateStarted(
+        uint64 newDepositDustThreshold,
+        uint256 timestamp
+    );
+    event DepositDustThresholdUpdated(uint64 depositDustThreshold);
+
+    event DepositTreasuryFeeDivisorUpdateStarted(
+        uint64 depositTreasuryFeeDivisor,
+        uint256 timestamp
+    );
+    event DepositTreasuryFeeDivisorUpdated(uint64 depositTreasuryFeeDivisor);
+
+    event DepositTxMaxFeeUpdateStarted(
+        uint64 newDepositTxMaxFee,
+        uint256 timestamp
+    );
+    event DepositTxMaxFeeUpdated(uint64 depositTxMaxFee);
+
+    event RedemptionDustThresholdUpdateStarted(
+        uint64 newRedemptionDustThreshold,
+        uint256 timestamp
+    );
+    event RedemptionDustThresholdUpdated(uint64 redemptionDustThreshold);
+
+    event RedemptionTreasuryFeeDivisorUpdateStarted(
+        uint64 newRedemptionTreasuryFeeDivisor,
+        uint256 timestamp
+    );
+    event RedemptionTreasuryFeeDivisorUpdated(
+        uint64 redemptionTreasuryFeeDivisor
+    );
+
+    event RedemptionTxMaxFeeUpdateStarted(
+        uint64 newRedemptionTxMaxFee,
+        uint256 timestamp
+    );
+    event RedemptionTxMaxFeeUpdated(uint64 redemptionTxMaxFee);
+
+    event RedemptionTimeoutUpdateStarted(
+        uint64 newRedemptionTimeout,
+        uint256 timestamp
+    );
+    event RedemptionTimeoutUpdated(uint64 redemptionTimeout);
```

Comment:
The type of `redemptionTimeout` is `uint32`:
```
uint32 redemptionTimeout;
```
Let's change it everywhere it occurs (functions, events).

Reply:
Done